### DISCUSSION TITLE
Feature Request: Exclude Covers

### DISCUSSION THREAD

**Discussion by brodkin** [jseidl/magic-areas#389] [Ideas / Feature Requests]
Hey there,

I'm running into a slight issue with my use case. I have a few rooms where there are "covers" of different types. The easier to explain are my dual roller shades, which include both a sun shade and blackout shade on each window. We generally only want to control the blackout shades, but the select list under "Basic area options" does not allow covers to be added for exclusion.

Since lights, sensors, media players and other domains supported are listed I am assuming this was an oversight.

Thank you for this project, which has significantly improved my HA install!

**Comment by jseidl** [comment]
Good call, will fix that in next update. Thanks for reporting and the kind words!
